<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestItemManager</class>
 <widget class="QWidget" name="TestItemManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="4">
    <widget class="QPushButton" name="selectFieldBtn">
     <property name="text">
      <string>选择领域</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>检测项目名称</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>别名</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>领域</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="OKBtn">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>缩写</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>限定范围</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="testNameEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>包含子参数</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>同时复制到其它领域</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="shortNameEdit"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="fieldBox"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="testAliasEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="selectItemBtn">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="clearItemBtn">
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="uniqueMarkEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
